docs/adr/0003-password-hashing-algorithm.md

```diff
@@ -1,7 +1,7 @@
 # Password hashing algorithm
 
-* Status: proposed
-* Date: 2022-12-23
+* Status: accepted
+* Date: 2023-10-25
```

Comment:
```suggestion
* Date: 2023-03-03
```
If I understand correctly (not 100% certain about that) this was actually decided when the PR was merged, we just forgot to mark it as accepted. If that is the case I would use that date.